PacketDefinitions420/PacketNotifier.cs

```diff
@@ -2329,5 +2329,46 @@ public void NotifyViewResponse(int userId, ViewRequest request)
 
             _packetHandlerManager.SendPacket(userId, answer, Channel.CHL_S2C, PacketFlags.None);
         }
+
+        /// <summary>
+        /// Notifies client that spell in <paramref name="slot"/> must be replaced with a <paramref name="newSpell"/>
+        /// </summary>
+        /// <param name="playerId">User to send the packet to.</param>
+        /// <param name="player">User to send the packe</param>
+        /// <param name="newSpell"></param>
+        /// <param name="slot"></param>
+        public void NotifySpellChangeResponse(IChampion player, string newSpell, byte slot)
+        {
+            var packet = new ChangeSlotSpellData_OwnerOnly
+            {
+                SenderNetID = player.NetId,
+                ChangeSpellData = new ChangeSpellDataSpellName
+                {
+                    IsSummonerSpell = slot == 4 || slot == 5,
+                    SpellName = newSpell,
+                    SpellSlot = slot,
+                }
+            };
+
+            _packetHandlerManager.SendPacket(player.PlayerId, packet.GetBytes(), Channel.CHL_S2C, PacketFlags.Reliable);
```

Comment:
`player.PlayerId` shouldn't be here. There is a way to go from Champion to userId with `PlayerManager`. The champion class shouldn't know about that.